bedrock/careers/tests/test_models.py

```diff
@@ -2,12 +2,17 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at https://mozilla.org/MPL/2.0/.
 
+from django.core.cache import cache
+
 from bedrock.careers.models import Position
 from bedrock.careers.tests import PositionFactory
 from bedrock.mozorg.tests import TestCase
 
 
 class TestPositionModel(TestCase):
+    def setUp(self):
+        cache.clear()
+
```

Comment:
for bonus points you could test that the db call doesn't get triggered on a 2nd call to the wrapped methods.

Reply:
Yeah - good call. Time to break out `assertNumQueries`!